### DISCUSSION TITLE
Request for Support for Heltec LoRa32 with SX1262

### DISCUSSION THREAD

**Discussion by MoonKnight171** [ExpressLRS/ExpressLRS#3054] [Q&A]
Hello ExpressLRS Team,

I hope this message finds you well. I am reaching out to express my interest in using the Heltec LoRa32, which features the ESP32-S3 microcontroller and the SX1262 radio chip.

I noticed that there is currently no support for the SX1262 in the ExpressLRS library, while there are drivers available for the SX127x and SX1280. I would like to understand the reasons behind this decision and whether there are any plans to include support for the SX1262 in the future.

I have attempted some modifications to get the Heltec LoRa32 working with ExpressLRS. I successfully got the OLED display to show the ExpressLRS logo, and the ExpressLRS TX Wi-Fi page is functioning as expected. However, I am encountering issues with the radio, as it displays a "bad radio" message.

Could you please provide guidance or assistance on how to make the SX1262 work with ExpressLRS? Any insights or recommendations would be greatly appreciated.

Thank you for your time, and I look forward to your response!

Best regards,  
MoonKnight171

**Comment by JyeSmith** [comment]
The SX1262 and LLCC68 were tested extensively 2 years ago and found to under perform compared to the SX1276.  So support was not added as it is a step backwards.

This info was posted on the Semtech forum and can be found on the Web Archive.

https://web.archive.org/web/20240310140117/https://forum.lora-developers.semtech.com/t/llcc68-poor-sensitivity/1480
